members/admin/user_admin.py

```diff
@@ -139,3 +142,24 @@ def get_list_filter(self, request):
                 AdminUserUnionListFilter,
                 AdminUserDepartmentListFilter,
             ]
+
+    def get_fieldsets(self, request, obj=None):
+        fieldsets = super(UserAdmin, self).get_fieldsets(request, obj)
+        if not obj:
+            return fieldsets
```

Comment:
Så vidt jeg kan gennemskue, kan vi starte med at checke for `obj is None`?
```suggestion
        if not obj:
            return fieldsets

        fieldsets = super(UserAdmin, self).get_fieldsets(request, obj)
```